MVR Docs (#293)
MystenLabs/suins-contracts: documentation/pages/move-registry/managing-package-info.mdx

    These examples assume that you use the MVR plugin for the Sui Typescript SDK, as described in the  [`Typescript SDK`](/move-registry/tooling/typescript-sdk) section.
</Callout>

> ## MVR name

Reviewer: Let's also mention that the standard supports an optional `/<version>` which allows specifying which on-chain version to resolve.

A MVR name consists of a SuiNS name and a package name, in the format `<suins_name>/<pkg_name>`.

> As an example, if your SuiNS name is `myname`, and your package name is `mypackage`,

Reviewer: ```suggestion
As an example, if your SuiNS name is `myname.sui`, and your package name is `mypackage`,
```

    These examples assume that you use the MVR plugin for the Sui Typescript SDK, as described in the  [`Typescript SDK`](/move-registry/tooling/typescript-sdk) section.
</Callout>

## MVR name

A MVR name consists of a SuiNS name and a package name, in the format `<suins_name>/<pkg_name>`.
As an example, if your SuiNS name is `myname.sui`, and your package name is `mypackage`,
the MVR name can be `@myname/mypackage` or `myname.sui/mypackage`.

> The MVR standard support an optional `/<version>` which allows specifying which on-chain version

Reviewer 2: ```suggestion
The MVR standard supports an optional `/<version>` that allows you to specify which on-chain version
```

```typescript
const transaction = new Transaction();
// ... (other code, could also include a `transaction.publish()`
// call to publish & register in one step.)

/// We pass in our UpgradeCap
const packageInfo = transaction.moveCall({
    target: `@mvr/metadata::package_info::new`,
    arguments: [transaction.object('<Your UpgradeCap (by value or object id)>')],
});

// We also need to create the visual representation of our "info" object.
// You can also call `@mvr/metadata::display::new` instead,
// that allows customizing the colors of your metadata object!
const display = transaction.moveCall({
    target: `@mvr/metadata::display::default`,
    arguments: [

>         transaction.pure.string('<Add a package name. This is not your MVR name.>') // Example: core

Reviewer: this is just a visual label to easily find these in the UI, let's not confuse them with the "package name" naming

    arguments: [transaction.object(packageInfo), display],
});

> // Set the default for the packageInfo, which enables reverse resolution

Reviewer: ```suggestion
// Set the default for the packageInfo, which enables reverse resolution for that network
```

// .. you can do any other actions, like setting the source code info, in the same PTB.
```

> ## Reverse Resolution

Reviewer: Let's also add that you might add a "default" on a "PackageInfo" object, but it needs to match what that name points to to resolve.

// .. you can do any other actions, like setting the source code info, in the same PTB.
```

> ## Reverse Resolution

Reviewer 2: ```suggestion
## Reverse resolution
```

Reverse resolution allows you to look up the human-readable name (like @suins/core or suins.sui/core)
associated with a given package ID, regardless of which version of the package you have.
This is useful for identifying and verifying packages on-chain, especially when working with tools or
interfaces that reference package IDs directly.

> To accomplish this you can add a `default` on a `PackageInfo` object (see example above), but it needs

Reviewer 2: ```suggestion
To accomplish this you can add a `default` on a `PackageInfo` object (see previous example), but it needs
```
